### PR TITLE
concat is working

### LINKED ISSUE #261: Create vertical concatenation primitive
We need a primitive that vertically concatenates the rows of input dataframes.  This is directly supported by Pandas via the dataframe's concat function:

https://pandas.pydata.org/docs/reference/api/pandas.concat.html

The primitive should take a list of dataframes as input, and return a single dataframe with all of the output.  The following hyperparams should be included:

`remove_duplicate_rows` - if there are two rows with the same `d3mIndex` only one should be retained.  Defaults to `True`.

`column_overlap` - can be one of `exact`, `union`, `intersection`.  For `exact`, columns have to be an exact match across all dataframes, with an error raised in the case they aren't. `union` takes all columns and fills in missing data with NaN, and `intersection`, which takes only columns that match across datasets, dropping others.  Should default to `union`.  The documentation for the `concat` function has examples for each case.

### PR DESCRIPTION
**Pull request by venkatkorapaty**
fixes #261

### CODE REVIEW

**Comment by venkatkorapaty** [comment]
On lines 43 and 81 I had to use `container.List`, so there's no type enforcement making sure that it's a list of `container.DataFrame`s. I don' think that will be a problem, however.

File: distil/primitives/concat.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+):
+    """
+    A primitive to encapsulate the functionality of pandas.concat.
+    """
+
+    metadata = metadata_base.PrimitiveMetadata(
+        {
+            "id": "b93e3e85-c462-4290-8131-abc51d76a6dd",
+            "version": version.__version__,
+            "name": "DistilRankedLinearSVC",
```
Comment (cdbethune) [inline distil/primitives/concat.py:54]: incorrect name

File: distil/primitives/concat.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            "python_path": "d3m.primitives.data_transformation.concat.VerticalConcatenationPrimitive",
```
Comment (cdbethune) [inline distil/primitives/concat.py:55]: Change to 

`d3m.primitives.data_transformation.concat.DistilVerticalConcat`

**Review by cdbethune — CHANGES_REQUESTED** [review]

File: distil/primitives/concat.py (hunk 1 above)
Comment (venkatkorapaty) [reply distil/primitives/concat.py:54]: Whoopsies, guess it's obvious where I copied the template from.

**Review by cdbethune — APPROVED** [review]

**Pull request merged**